src/app/drive/components/DriveExplorer/DriveExplorer.tsx

```diff
@@ -959,9 +959,7 @@ const uploadItems = async (props: DriveExplorerProps, rootList: IRoot[], files:
             onSuccess: onDragAndDropEnd,
           },
         }));
-        dispatch(storageThunks.uploadMultipleFolderThunkNoCheck(folderDataToUpload)).then(() => {
-          dispatch(fetchSortedFolderContentThunk(currentFolderId));
```

Comment:
Is this no longer necessary for the folder to be displayed in order?

Reply:
Yes it is needed but I had to move it inside uploadMultipleFolderThunkNoCheck